wp-content/themes/workingnyc/assets/js/employer-program-single-page-development.js

```diff
@@ -0,0 +1,31 @@
+(function () {
```

Comment:
The animation shows all three cards moving but the cards' positions only change by 1 (when you click the arrow, it looks like it goes from [card 1] [card 2] [card 3] to [card 4] [card 5] [card 6], but it actually goes to [card 2] [card 3] [card 4]). You should keep the cards as [card 2] [card 3] [card 4] but change the animation so it looks like that is what's happening

https://github.com/CityOfNewYork/working-nyc/assets/8957837/95f793bc-7017-44d1-82ef-5ccafc50323b